debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/Record.java

```diff
@@ -12,20 +12,29 @@ public class Record {
     public String snapshot;
     public String op;
     public long vsn; // Voyager Sequence Number.
-    public ArrayList<String> keyColumns = new ArrayList<>();
-    public ArrayList<Object> keyValues = new ArrayList<>();
-    public ArrayList<String> valueColumns = new ArrayList<>();
-    public ArrayList<Object> valueValues = new ArrayList<>();
+
+     // Value information for 'before' struct
+    public ArrayList<String> beforeValueColumns = new ArrayList<>();
+    public ArrayList<Object> beforeValueValues = new ArrayList<>();
+
+    // Key and value information for 'after' struct
+    public ArrayList<String> afterKeyColumns = new ArrayList<>();
```

Comment:
There is only a single key, so you can continue calling  this `keyColumns` and `keyValues`

Reply:
got it.